impeller/entity/entity_unittests.cc

```diff
@@ -2097,11 +2097,11 @@ TEST_P(EntityTest, RuntimeEffect) {
     contents->SetRuntimeStage(runtime_stage);
 
     struct FragUniforms {
-      Scalar iTime;
       Vector2 iResolution;
```

Comment:
By re-arranging these uniforms we can observe that the shader does not work without this fix. Thus, tested.

Reply:
this doesn't animate without the fix, because the first uniform gets data in 0, 1, and the second uniform gets the fixed data in 1 instead of at 2.